symfony/phpunit-bridge/5.3/manifest.json

```diff
@@ -6,7 +6,6 @@
         "tests/": "tests/"
     },
     "gitignore": [
-        ".phpunit",
```

Comment:
Why `.phpunit` has been removed from .gitignore? I don't think that directory should be committed to source code, in my case it contains 2600+ files...

Reply:
Because now we prefer using the binary from PHPUnit (which does not create `.phpunit` folder) instead the binary from Symfony PHPUnit bridge (`simple-phpunit`).

So adding `.phpunit` in `.gitignore` is not useful anymore.